### ISSUE TITLE
Unable into import FormControlStatus

### ISSUE THREAD

**Issue by penfold** [angular/angular#44176]
### Which @angular/* package(s) are the source of the bug?

forms

### Is this a regression?

No

### Description

I have a method like:

```
protected waitForAsyncValidators$(): Observable<FormControlStatus> {
        return this.form.statusChanges
            .pipe(
                startWith(this.form.status),
                tap(status => {
                    this.logService.debug(`Form status: ${status}`);
                }),
                filter(status => status !== FormControlStatus.Pending),
                take(1)
            );
    }
```
That is being reworked for ng 13 using the new FormControlStatus type.

I'm unable to import the FormControlStatus type. Where am I going wrong?

### Please provide the exception or error you saw

```true
error TS2304: Cannot find name 'FormControlStatus'.

filter(status => status !== FormControlStatus.Pending),
```

### Please provide the environment you discovered this bug in

```true
Angular 13.0.x
```

**Comment by JoostK** [comment]
This is a bug. The `@angular/forms` package should've exported `FormControlStatus` from its public entry-point, but it does not.

**Comment by JoostK** [comment]
That said; `FormControlStatus` is only a type, not an enum! So in your case you'll have to continue using the string literal `'PENDING'`.

**Comment by AndrewKushnir** [comment]
@penfold thanks for reporting the issue, we'll make the `FormControlStatus` symbol available via #44183.

**Comment by penfold** [comment]
@AndrewKushnir Great work. Thanks.

**Comment by AndrewKushnir** [comment]
Thanks for reporting the issue @penfold, the fix would be available in v13.0.3 (in a week from now).
